chatkit/src/main/java/com/stfalcon/chatkit/messages/MessageTextUtils.kt

```diff
@@ -129,19 +174,21 @@ class MessageTextUtils {
 
     data class PatternSpanDescriptor(val startIndex: Int, val endIndex : Int, val content: String, val label: String? = null, val isLink: Boolean = true, val isBold: Boolean = false, val isItalic: Boolean = false, val isStroke: Boolean = false)
 
-    data class PatternDescriptor(val content: String, val label: String? = null, val isLink: Boolean = true, val isBold: Boolean = false, val isItalic: Boolean = false, val isStroke: Boolean = false){
+    data class PatternDescriptor(var content: String, val label: String? = null, var isLink: Boolean = true, var isBold: Boolean = false, var isItalic: Boolean = false, var isStroke: Boolean = false, var surrounding: MarkDown = MarkDown.NONE, var beginIndex: Int = 0, var endIndex: Int = 0){
         fun toTag() : String{
+            if(content.contains("|"))
+                content = content.split("|")[1]
             return when {
```

Comment:
expresion can be simmplified to

`            return when (surrounding) {
                MarkDown.LINK -> {
                    var swapData = "<$content"
                    label?.run {
                        swapData += "|$label"
                    }
                    swapData + ">"
                }
                MarkDown.BOLD -> "*$content*"
                MarkDown.ITALIC -> "_${content}_"
                MarkDown.STROKE -> "~$content~"
                else -> content
            }`